### ISSUE TITLE
TagGroup: focus ring does not use Highlight color with forced-colors: active

### ISSUE THREAD

**Issue by majornista** [adobe/react-spectrum#4197]
# 🐛 Bug Report 

TagGroup: focus ring does not use Highlight color with forced-colors: active

## 🤔 Expected Behavior

Both the border-color and focus ring box-shadow color should be overridden to use Highlight with forced-colors: active.

## 😯 Current Behavior

The following screen shot shows a TagGroup with forced-colors: active. The focused Tag uses Highlight as a border-color, but the focus ring box-shadow still uses the blue focus ring color from Spectrum.

<img width="721" alt="TagGroup with forced-colors: active" src="https://user-images.githubusercontent.com/154077/225019707-943fee4f-cf02-4a5e-8970-d95d0b277c9b.png" />

## 🔦 Context

FY23 Accessibility Audit

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| react-spectrum   |  @adobe/react-spectrum@3.25.1
| Browser          |  Google Chrome Version 110.0.5481.177 (Official Build) (arm64)
| Operating System | macOS Ventura 13.2.1 (22D68)

## 🧢 Your Company/Team

Adobe/Accessibility

## 🕷 Tracking Issue (optional)

A11Y-5870
